Sources/Core/Utility/Array+Extension.swift

```diff
@@ -106,3 +106,12 @@ extension Array where Element: BinaryInteger {
         return sorted_data[index]
     }
 }
+
+//MARK: - Conversion
```

Comment:
Add a space between `//` and `MARK`.

Reply:
good catch! 
It would be nice to have swiftlint -> https://github.com/web3swift-team/web3swift/issues/436
If I can help with that let me know :)

Reply:
We already have swiftlint. [See this file](https://github.com/web3swift-team/web3swift/blob/develop/.swiftlint.yml).
Just not sure about the rule for this type of formatting issue.

Reply:
It's just not part of the CI yet.

Reply:
apologies again, I understood the issue wrong.